src/calculate_fusion.c

```diff
@@ -0,0 +1,353 @@
+#include <stdio.h>
+#include <math.h>
+#include <stdlib.h>
+#include <string.h>
+#include <gsl/gsl_math.h>
+#include <gsl/gsl_eigen.h>
+#include "calculate_fusion.h"
+
+struct support_degree_matrix *calculate_support_degree_matrix(void)
+{
+    // if (node == NULL)
+    // {
+    //     printf("%s: Incorrect Input\n", __func__);
+    //     return;
+    // }
+    // int i = 0;
+    // while (node != NULL)
+    // {
+    //     printf("%c\n", node->sensor_name);
+    //     node = node->next;
+    //     i++;
+    //     printf("%d\n", i);
+    // }
+    float values[] = {10, 0};
```

Comment:
is this for testing?

Reply:
since this function currently does not take any values, id assume so. In any case this needs to be adjusted